src/core/connection.h

```diff
@@ -1067,6 +1067,10 @@ QuicConnRelease(
     QuicConnValidate(Connection);
 
 #if DEBUG
+    if (Connection->RefTypeCount[Ref] <= 0) {
+        fprintf(stderr, "RefTypeCount[%d] is %d\n", Ref, Connection->RefTypeCount[Ref]);
```

Comment:
`RefTypeCount[6] is -22829`

RefTypeCount is `short` type (from -32,768 to 32,767). So not just off-by-one issue

Reply:
OSS-Fuzz issue dashboard has been listing same issue and the cause seems to be from other PR?

https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=68176&q=msquic
https://oss-fuzz.com/testcase-detail/4533150459756544
https://github.com/microsoft/msquic/compare/d65bb1af4cc96f996d0312f445ce648f114b380d...102a491ff1394a1bad4d98350126ec2681814700

Reply:
@nibanks
Reverting https://github.com/microsoft/msquic/pull/3931 solves CIFuzz issue which has also been reported from OSS-Fuzz before integrating priority feature.
The reason only CIFuzz and OSS-Fuzz find should be due to its random algorithm based on genetic algorithm.
Do you have some idea the change has potential timing issues?

Reply:
My guess is that the combination of the that change and priority exposes some bug, most likely in spinquic, for object lifetime. But I have no idea specifically what could be causing it.